kythe/java/com/google/devtools/kythe/extractors/java/JavaCompilationUnitExtractor.java

```diff
@@ -498,6 +498,9 @@ private void processRequiredInput(
     if ((isJarPath && jarPath.startsWith(jdkJar)) || path.startsWith(JDK_MODULE_PREFIX)) {
       return;
     }
+    if (path.contains("Generated.class")) {
```

Comment:
endsWith("javax/annotation/Generated.class")?

Reply:
Oops, that was for debugging :-( I'll delete it.